place_tracker/lib/place_util.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'place.dart';
+
+class PlaceUtil {
+  static BitmapDescriptor getPlaceMarkerIcon(PlaceCategory category) {
+    // TODO(kenzieschmoll): use custom marker assets.
+    switch (category) {
+      case PlaceCategory.favorite:
+        return BitmapDescriptor.defaultMarkerWithHue(BitmapDescriptor.hueRed);
+      case PlaceCategory.visited:
+        return BitmapDescriptor.defaultMarkerWithHue(
+            BitmapDescriptor.hueViolet);
+      case PlaceCategory.wantToGo:
+      default:
+        return BitmapDescriptor.defaultMarkerWithHue(BitmapDescriptor.hueAzure);
+    }
+  }
+
+  static List<Place> getPlacesForCategory(
```

Comment:
Looks like this could be a private method on PlaceMap (and it's always applied to _selectedCategory).